etest/tests.py

```diff
@@ -91,7 +90,7 @@ def environment(self):
         if "python" in self.ebuild.compat:
             _["PYTHON_TARGETS"] = " ".join(self.ebuild.compat["python"])
 
-            # Things still want python2…☹
+            # Things still want python2.7
```

Comment:
They do?  Wow.

Reply:
You added that before I even got to touch it :p I just fixed a random utf spill. I think python2 was removed from ::gentoo a while ago, might want to revisit this.

Reply:
What do you mean a random utf spill?  I bet this is from ancient commits other than black formatting.  Removing python 2 makes sense and should probably be another issue but we can also leave it if it's not harmful.

Reply:
I now see it was supposed to be an emoji. Whoops. It showed up as a random mess of characters in vim :/

Reply:
Hmm, does your vim not have unicode support?  That seems like a problem with environment rather than the code to me. ;)

Reply:
I think its a problem on my console, I have my locale set as UTF8 anyways.